src/App.js

```diff
@@ -3,14 +3,13 @@ import Header from "./components/Header";
 import SidePanel from "./components/SidePanel";
 import MainContent from "./components/MainContent";
 import "./App.css";
-// import MarketCapFetch from "./utils/Api2";
 
 function App() {
     return (
         <main>
             <Header />
             <MainContent />
-            {/* <MarketCapFetch /> */}
+
```

Comment:
In the next branch, remove this extra space. No point for empty lines anywhere like this.